test/python/sdpa/frost/test_sdpa_fwd_fp8_sm100.py

```
@@ -249,6 +256,58 @@ def test_fp8_padding(in_key, causal):
 
 
+@pytest.mark.L0
+@pytest.mark.parametrize("in_key", _INS)
+@torch_fork_set_rng(seed=0)
+def test_fp8_stats_less_zero_workspace(in_key):
+    """No Stats output: the kernel compiles the LSE store out (has_lse=False),
+    no dummy buffer exists at any level, and the dense graph reports
+    ``get_workspace_size() == 0`` (asserted inside ``_run``). The Amax_S /
+    Amax_O atomicMax writes are independent of the LSE and still produced."""
+    scale = 1.0 / math.sqrt(128)
+    O, O_ref, a_s, a_s_ref, a_o, a_o_ref = _run(2, 8, 8, 256, 256, in_key, torch.float16, scale=scale, sdpa_kwargs=dict(use_causal_mask=True), stats=False)
```

Ruff (0.16.1)

[error] Ambiguous variable name: `O`

(E741)